docs/application-development-guide.md

```diff
@@ -3132,6 +3132,55 @@ const handle = await client.start(scheduledWorkflow, {
 
 TODO
 
+<Tabs
+defaultValue="go"
+groupId="site-lang"
+values={[{label: 'Go', value: 'go'},{label: 'Java', value: 'java'},{label: 'PHP', value: 'php'},{label: 'Typescript', value: 'typescript'},]}>
+
+<TabItem value="go">
+
+Content is not available
+
+</TabItem>
+<TabItem value="java">
+
+Content is not available
+
+</TabItem>
+<TabItem value="php">
+
+To create a local Activity, use `#[LocalActivityInterface]`.
+
+```php
+use Temporal\Activity\LocalActivityInterface;
+
+#[ActivityInterface]
+interface YourActivityInterface
+{
+    #[ActivityMethod("youractivity")]
+    public function youractivity(): string;
+}
+```
+
+Local activity requires `LocalActivityOptions`:
+
+```php
+$greetingActivity = Workflow::newActivityStub(
+    YourActivityInterface::class,
+    LocalActivityOptions::new()->withStartToCloseTimeout(\DateInterval::createFromDateString('30 seconds'))
+);
+```
+
+Local Activities have a limited set of options, including: `ScheduleToCloseTimeout`, `StartToCloseTimeout`, `RetryPolicy`.
+
+</TabItem>
+<TabItem value="typescript">
+
+Content is not available
+
+</TabItem>
+</Tabs>
+
 ## Observability
 
 TODO
```

Comment:
If this is going to stay in the release, change it to something like:

"This section is in progress."

Something more than just TODO

Reply:
This goes for all TODOs later on

Reply:
Ya, I agree with that.
I have a bad habit of leaving "todo" in places.
I will clean up my act on this too.